### ISSUE TITLE
Hook breaks with Thermos loadedTileEntityList cleanup

### ISSUE THREAD

**Issue by LunNova** [MinimallyCorrect/TickProfiler#33]
TCPR/Thermos/issues/52

Not sure exactly what the problem is.

**Comment by LunNova** [comment]
Reports of KCauldron + TickProfiler causing entities to stop ticking.

I don't know which builds of either mod are being used. I have not received any reports here of the issue :/

**Comment by LunNova** [comment]
Triggering this issue would result in this error:

```
Log.error("Looks like another mod broke TickProfiler's replacement entity list in world: " + Log.name(world));
```

**Comment by LunNova** [comment]
Issue only occurs with Thermos, not KCauldron.

I will resolve this by dropping the hooking via entity lists and using a less fragile implementation with ModPatcher.

**Comment by LunNova** [comment]
Fixed in 616fb949ea2e9c1484dff
